Makefile

```diff
@@ -352,12 +368,12 @@ swagger_spec: SHELL:=$(RUN_IN_DEV_SHELL)
 swagger_spec: images/dev-env/.dockerbuilt
 	swag init -g cmd/chaos-dashboard/main.go --output pkg/dashboard/swaggerdocs --pd --parseInternal
 
-.PHONY: all clean test install manifests groupimports fmt vet tidy image \
-	docker-push lint generate config \
-	install.sh $(GO_TARGET_PHONY) \
-	manager chaosfs chaosdaemon chaos-dashboard \
-	dashboard dashboard-server-frontend gosec-scan \
-	failpoint-enable failpoint-disable swagger_spec \
+.PHONY: all image clean test install manifests manifests/crd.yaml \
```

Comment:
I also arranged and removed some phony targets, below are targets that have been removed because they are outdated:

- manager
- chaosfs
- chaosdaemon
- chaos-dashboard
- dashboard
- dashboard-server-frontend
- go_build_cache_directory